packages/augur-core/source/contracts/utility/Formulas.sol

```diff
@@ -0,0 +1,62 @@
+pragma solidity 0.5.4;
+
+import 'ROOT/reporting/IDisputeWindow.sol';
+import 'ROOT/libraries/math/SafeMathUint256.sol';
+import 'ROOT/reporting/Reporting.sol';
+
+
+/**
+ * @title Formulas
+ * @notice A Utility contract with no state meant to hosue the logic for large formulas which don't fit into alreayd large contracts
+ */
+contract Formulas {
+    using SafeMathUint256 for uint256;
+
+    function calculateValidityBond(IDisputeWindow  _previousDisputeWindow, uint256 _previousValidityBondInAttoCash) public view returns (uint256) {
```

Comment:
might be a good idea to write out what the formulas are suppose to to. There will be lots of questions on how the different bonds float.